modules/cosmos/src/models/chain.ts

```diff
@@ -0,0 +1,9 @@
+import { Account } from "@firewatch/core/account";
+import { Chain } from "@firewatch/core/chain";
+import { CosmosModule } from "../models/module";
+
+export interface CosmosChain extends Chain {
+    height: number;
+    module: CosmosModule;
+    account: Pick<Account, "name">;
```

Comment:
Perque ha de ser un objecte i no un string?

Estaria be que cada submodule de cosmos tingui la seva address a la config i no tindre una per a tot